src-docs/src/views/form_layouts/accessible_labels.js

```diff
@@ -13,51 +12,39 @@ export default () => {
 
   return (
     <EuiForm component="form">
-      <EuiDescribedFormGroup
-        title={<h3>Not as simple as they look</h3>}
-        description={
+      <EuiFormRow
+        label="Settings"
+        helpText={
           <>
-            Some form controls come with their own label and don&lsquo;t need
-            the one provided by <strong>EuiFormRow</strong>. Include{' '}
-            <EuiCode>hasChildLabel:false</EuiCode> on form rows that wrap:{' '}
-            <EuiCode>EuiSwitch</EuiCode>, <EuiCode>EuiButton</EuiCode>, or{' '}
-            <EuiCode>EuiLink</EuiCode>.
+            Navigate to this Switch with a screen reader. With{' '}
+            <EuiCode>hasChildLabel = false</EuiCode> the name of this is
+            &ldquo;Dark mode?&rdquo; instead of &ldquo;Settings&rdquo;.
+          </>
+        }
+        hasChildLabel={false}>
+        <EuiSwitch
+          name="switch"
+          label="Dark mode?"
+          onChange={() => {
+            setCheckedState(false);
+            setTimeout(() => {
+              setCheckedState(true);
+            }, 500);
+          }}
+          checked={checkedState}
+        />
+      </EuiFormRow>
+      <EuiFormRow
+        label="Your cluster data"
+        helpText={
+          <>
+            Navigate to this Switch with a screen reader. With{' '}
+            <EuiCode>hasChildLabel = true</EuiCode> the name of this switch is
+            &ldquo;Your cluster data&rdquo; instead of &ldquo;Download&rdquo;.
           </>
         }>
```

Comment:
One thing to note, is that you are using this as an example of what  **not** to do, but since it's just copy/paste code, people may not read that. We tend to always give the "right" way in code snippets/examples leaving the do/don't type of examples to the guidelines pages.

My suggestion is to switch this to show the **right** usage.

Reply:
We show the right usage directly above so I wonder how much value there is in showing the same thing again?

Should I just remove the bad example?
Or maybe add a code comment to help highlight that it's a negative example in case folks copy-and-paste? (The snippet, which I imagine gets more copy-and-paste usage, is showing the correct example.)

Reply:
I think both examples are valid to showcase (switch vs button), but I do think the examples should be the **correct** usage, even if they are the same for both instances.

Reply:
Consumers are most likely just going to look for the similar implementation to theirs, and not read the text necessarily. So if they have a button inside a form row, they'll just copy that example without reading.